docs: add validation disclaimer (#53)
axa-group/bauta.js: packages/bautajs-express/README.md

```

See a full example on [Example of a project from scratch](../../docs/hello-world.md).

### Validation

> Fastify has a out-of-the-box request validation, therefore the builtin Bauta.js request validations is disabled on this plugin.

Reviewer: Fastify has an out-of-the-box request validation, therefore the builtin Bauta.js request validations are disabled on this plugin.

> Fastify has a out-of-the-box request validation, therefore the builtin Bauta.js request validations is disabled on this plugin.

Reviewer: ```suggestion
Fastify has an out-of-the-box request validation, therefore the builtin Bauta.js request validations are disabled on this plugin.
```
